arches/app/views/graph.py

```diff
@@ -422,13 +422,20 @@ def post(self, request, graphid=None):
                     clone_data = graph.copy(root=data)
                     clone_data["copy"].slug = None
                     clone_data["copy"].save()
+
                     ret = {"success": True, "graphid": clone_data["copy"].pk}
 
                 elif self.action == "clone_graph":
                     clone_data = graph.copy()
                     ret = clone_data["copy"]
                     ret.slug = None
+                    ret.publication = None
+
                     ret.save()
+
+                    if bool(graph.publication_id):
+                        ret.publish(user=request.user)
```

Comment:
👍 

I just had a quick look at save() and publish() and it doesn't look like we're in a scenario where need to refresh_from_db() before doing this. (Whereas if this were an add_node() operation that created node aliases in postgres on save, I could see that maybe being a thing to be careful about.)